lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-lenovo-physical_infra_manager-physical_server.rb.rb

```diff
@@ -0,0 +1,4 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Lenovo_PhysicalInfraManager_PhysicalServer < MiqAeServicePhysicalServer
+  end
```

Comment:
@AndreyMenezes Please verify that you are running with this code change: https://github.com/ManageIQ/manageiq/pull/14465.